### PR TITLE
Remove TimingContext repr

### PR DESCRIPTION
**Pull request by gbova**
https://github.com/dimagi/commcare-hq/pull/25106#pullrequestreview-276676713

### CODE REVIEW

**Review by esoergel — APPROVED** [review]

File: corehq/util/timer.py [hunk 1]
```diff
@@ -201,12 +201,6 @@ def to_list(self, exclude_root=False):
         """Get the list of ``NestableTimer`` objects in hierarchy order"""
         return self.root.to_list(exclude_root)
 
-    def __repr__(self):
-        return "TimingContext(root='{}')".format(
-            self.root,
-            self.root.to_dict()
```
Comment (millerdev) [inline corehq/util/timer.py]: Hmm, I was only suggesting to remove this one line (`self.root.to_dict()`), not the entire `__repr__` function, which I think is useful to keep around for debugging. Sorry for the not-very-clear https://github.com/dimagi/commcare-hq/pull/25106#pullrequestreview-276676713.

Comment (gbova) [reply corehq/util/timer.py]: Gotcha! That makes a lot of sense

**Review by millerdev — APPROVED** [review]

**Review by kaapstorm — APPROVED** [review]
(One more day to go, and then we can use f-strings.)

**Comment by orangejenny** [comment]
> (One more day to go, and then we can use f-strings.)

Glad I'm not the only one counting down...one more day and I can merge some tests with py2 encoding problems.

**Pull request merged**